gpu/ipc/service/gpu_init.cc

```diff
@@ -891,7 +891,11 @@ bool GpuInit::InitializeAndStartSandbox(base::CommandLine* command_line,
       std::move(supported_buffer_formats_for_gl_native_pixmap_import);
   [[maybe_unused]] auto* factory =
       ui::OzonePlatform::GetInstance()->GetSurfaceFactoryOzone();
+#if BUILDFLAG(IS_COBALT)
```

Comment:
This looks .. not great just to avoid an unused var. Maybe we can do an override in GN, or perhaps better to use 

```
#if !BUILDFLAG(ENABLE_VULKAN) && !BUILDFLAG(SKIA_USE_DAWN) 
```

rather here - the latter would be upstreamable ?